grader/settings.py

```diff
@@ -133,7 +133,7 @@
 ##########################################################################
 CONTAINER_MODE = False
 CONTAINER_SCRIPT = join(BASE_DIR, "scripts/docker-run.sh")
-CELERY_BROKER = False #'amqp://guest@localhost/'
```

Comment:
`CONTAINER_MODE` must be removed since we don't support any other mode any more. Setting it to False would not change anything. `CONTAINER_SCRIPT` is still used and preserved.

I still think that Celery and RabbotMQ settings are not used outside the chroot sandbox, so you need to tell us if there is any real reason to keep them:

* CELERY_BROKER
* RABBITMQ_MANAGEMENT
* CELERY_TASK_LIMIT_SEC
* CELERY_TASK_KILL_SEC
* CELERY_AFFINITIES

You should check if `QUEUE_ALERT_LENGTH` has any real effect with the container grading.

Reply:
Done!